website/content/docs/configuration-options.md

```diff
@@ -3,13 +3,14 @@ title: Configuration Options
 group: reference
 weight: 23
 ---
+
```

Comment:
Can you revert the format changes to this file and only add the `hidden` docs changes?
It makes it very hard to know what had changed.

Reply:
I only added [hidden property in line 200](https://github.com/netlify/netlify-cms/pull/3643/files#diff-1b4d596ba92ef875ca90e980db6f0b98R200) but the prettier formatter changed the format of the document. I'll revert it back.